packages/server/core/src/base/middlewares/renderHandler/render.ts

```diff
@@ -1,11 +1,11 @@
 import type { IncomingMessage } from 'http';
 import { ServerRoute } from '@modern-js/types';
+import { cutNameByHyphen } from '@modern-js/utils';
```

Comment:
We should not import from '@modern-js/utils'
`@modern-js/utils` can't run in worker env

Reply:
Got, let me modify it

Reply:
done